Find a functionally equivalent code:com.android.internal.os.ZygoteInit.runSelectLoop:COMMENT
Method Modifier: private     static      internal    
Comment:/**
 * Runs the zygote process's select loop. Accepts new connections as
 * they happen, and reads commands from connections one spawn-request's
 * worth at a time.
 *
 * @throws MethodAndArgsCaller in a child process when a main() should
 * be executed.
 */

@@ -31,39 +31,39 @@
         if (index < 0) {
             throw new RuntimeException("Error in select()");
         } else if (index == 0) {
-            ZygoteConnection newPeer = acceptCommandPeer();
+            ZygoteConnection newPeer = acceptCommandPeer(abiList);
             peers.add(newPeer);
-            fds.add(newPeer.getFileDesciptor());
+            fds.add(newPeer.getFileDescriptor());
         } else {
             boolean done;
             done = peers.get(index).runOnce();

